src/style.css

```diff
@@ -0,0 +1,11 @@
+body {
+    /* background-color: #F1EFEF; */
+    background-image: url(./img/stockvault-abstract-blurred-beautiful-soft-pink-cloud-background268659.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.MuiFormControl-root {
+    width: 100%;
+} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693537)